### ISSUE TITLE
Eliminate the existing typos

### ISSUE THREAD

**Issue by zyy17** [GreptimeTeam/greptimedb#589]
### What type of enhancement is this?

Refactor

### What does the enhancement do?

The current project exist some typos, we can
- [x] use the [typos](https://github.com/crate-ci/typos) to find out all of the typos and fix them in one PR. #592 
- [x] After that, we can add a typo [checker](https://github.com/marketplace/actions/typos-action) in GitHub Action and check for every new PR.

**Comment by dongxuwang** [comment]
I'm willing to do the first half.

**Comment by zyy17** [comment]
@dongxuwang Thank you for your involvement 😊. I will assign the issue to you.

**Comment by Adejuwon-00** [comment]
I'll be glad to chip in

**Comment by zyy17** [comment]
@Adejuwon-00 Since @dongxuwang already is working on fixing the existing typo, maybe you can help us to add the GitHub action after #592 is merged.

**Comment by dongxuwang** [comment]
I guess this issue can be closed.

**Comment by zyy17** [comment]
@dongxuwang Thank you for your PRs! 😊
